test/unit_tests/modules/util/data-set-spec.js

```diff
@@ -1,11 +1,52 @@
 'use strict';
 
 var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+var dataSet = require( BASE_JS_PATH + 'modules/util/data-set' ).dataSet;
 
 var chai = require( 'chai' );
 var expect = chai.expect;
-var dataSet = require( BASE_JS_PATH + 'modules/util/data-set' );
+var jsdom = require( 'mocha-jsdom' );
+var sinon = require( 'sinon' );
+var sandbox;
+var baseDom;
+
+var HTML_SNIPPET =
+  '<div data-test-value-a="testValueA"' +
+        'data-test-value-B="testValueB"' +
+        'data-testValue-C="testValueC"' +
+        'data-test-ValuE-D="testValueD"' +
+        'data-TEST-value-E="testValueE" >' +
```

Comment:
Should the `data-test-value-*` formats all be different? Is that part of what's tested? In particular, should L:16 be missing the dash?